lib/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I added this new file. This will be used to check the js files inside the `lib` folder. It won't emit any javascript files

Reply:
great, thanks for the heads up